### PR TITLE
Fixes Email Clients not generating logins/AIC eye toggle

### PR DESCRIPTION
**Pull request by TerryTehCoder**
## About the Pull Request

Fixes the email clients not properly generating logins/usernames which prevented accounts from being created properly at the beginning of the round.

Adds in an AIC Eye Toggle that lets you disable the A.I face that indicates your center of view. It's still there, it just makes the Icon disappear. It can be toggled back on by pressing the command again.

Disables the NTOS computer configuration tool from the subsystems tab. 
It was added when DTraitor fixed everything up and added the base nano_modules to the list.

## Why It's Good For The Game

Bugfixes are lovely!

The AIC Eye always bothered me and I think having the option to toggle it without explicitly turning it off is great for anything that's not mechanically impacting the round.

## Changelog

:cl:
fix: Fixed email clients not properly generating login/username at round start.
fix: Removes Computer Configuration Tool from subsystems tab.
Add: AIC Eye Toggle
/:cl:

### CODE REVIEW

**Comment by MACIEKBAKI** [comment]
you might want to include a proper changelog

File: code/modules/mob/observer/freelook/ai/eye.dm [hunk 1]
```diff
@@ -106,3 +106,16 @@
 
 	eyeobj.acceleration = !eyeobj.acceleration
 	to_chat(usr, "Camera acceleration has been toggled [eyeobj.acceleration ? "on" : "off"].")
+
+/mob/living/silicon/ai/proc/eyepuppettoggle() //Messing with the SEE_INVISIBLE_TAG renders the lighting differently so I'd rather just swap out the icon for nothing and still render lights correctly.
```
Comment (MACIEKBAKI) [inline code/modules/mob/observer/freelook/ai/eye.dm]: `/mob/living/silicon/ai/proc/ai_eye_puppet_toggle()` or `/mob/living/silicon/ai/proc/eye_puppet_toggle()` would be preferable as it keeps the naming scheme

File: code/modules/mob/living/silicon/ai/ai.dm [hunk 2]
```diff
@@ -32,7 +32,8 @@ var/list/ai_verbs_default = list(
 	/mob/living/silicon/ai/proc/show_crew_manifest,
 	/mob/living/silicon/ai/proc/ai_power_override,
 	/mob/living/silicon/ai/proc/ai_shutdown,
-	/mob/living/silicon/ai/proc/ai_reset_radio_keys
+	/mob/living/silicon/ai/proc/ai_reset_radio_keys,
+	/mob/living/silicon/ai/proc/eyepuppettoggle
```
Comment (MACIEKBAKI) [inline code/modules/mob/living/silicon/ai/ai.dm]: of course youd change here as necessary

File: code/modules/tgui/modules/ntos-only/configurator.dm [hunk 3]
```diff
@@ -2,6 +2,7 @@
 	name = "NTOS Computer Configuration Tool"
 	ntos = TRUE
 	tgui_id = "Configuration"
+	available_to_ai = FALSE
```
Comment (MACIEKBAKI) [inline code/modules/tgui/modules/ntos-only/configurator.dm:5]: stealth nerf???

**Comment by TerryTehCoder** [comment]
> you might want to include a proper changelog
> 
> 

My changelog was deleted in a prior upload. Thank you for pointing it out.

Updated to include changelog.

File: code/modules/tgui/modules/ntos-only/configurator.dm (hunk 3 above)
Comment (TerryTehCoder) [reply code/modules/tgui/modules/ntos-only/configurator.dm:5]: This is mentioned in the PR body and the changelog that got deleted.

Comment (MACIEKBAKI) [reply code/modules/tgui/modules/ntos-only/configurator.dm:5]: alright but why remove it? wouldn't it make sense the AI can do computer config?

Comment (TerryTehCoder) [reply code/modules/tgui/modules/ntos-only/configurator.dm:5]: This is a hardware configuration in the AIC subsystems tab. Which would be fine if the AIC actually had any hardware to configure in that sense.  It doesn't. 

I don't believe this program is accessible in the general programs list either?

Comment (MACIEKBAKI) [reply code/modules/tgui/modules/ntos-only/configurator.dm:5]: ah, that'd make sense, yeah; confused it for general network config

**Review by hry-gh — APPROVED** [review]

**Pull request merged**